platform/android/java/lib/patches/com.google.android.vending.licensing.patch

```diff
@@ -6,9 +6,9 @@ index 7c42bfc28..feb579af0 100644
      public void putString(String key, String value) {
          if (mEditor == null) {
              mEditor = mPreferences.edit();
-+            // -- GODOT start --
++            // -- BLAZIUM start --
 +            mEditor.apply();
```

Comment:
This line maybe from a different branch, please validate they are required for rebranding